src/libraries/System.Runtime/tests/System/Text/ASCIIUtilityTests.cs

```diff
@@ -449,7 +449,7 @@ public UnsafeLazyDelegate(string methodName)
 
                     // Construct the TDelegate pointing to this method
 
-                    return (TDelegate)Activator.CreateInstance(typeof(TDelegate), new object[] { null, methodInfo.MethodHandle.GetFunctionPointer() });
+                    return (TDelegate)(object)methodInfo.CreateDelegate(typeof(TDelegate));
```

Comment:
```suggestion
                    return methodInfo.CreateDelegate<TDelegate>();
```
And also change the constrain above to System.Delegate